core/src/cluster_info.rs

```diff
@@ -2670,13 +2159,16 @@ mod tests {
     }
 
     fn test_split_messages(value: CrdsValue) {
-        const NUM_VALUES: usize = 30;
+        const NUM_VALUES: u64 = 30;
         let value_size = value.size();
-        let expected_len = NUM_VALUES / (MAX_PROTOCOL_PAYLOAD_SIZE / value_size).max(1) as usize;
-        let msgs = vec![value; NUM_VALUES];
+        let num_values_per_payload = (MAX_PROTOCOL_PAYLOAD_SIZE / value_size).max(1);
```

Comment:
@sagar-solana on `test_split_messages_large`, `MAX_PROTOCOL_PAYLOAD_SIZE / value_size == 0`, does that mean there are just some messages that are too large to fit in the payload?

Reply:
Yes, things like epoch_slots can grow too large.